src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +51,161 @@ export const KafkaPage: FunctionComponent = () => {
+                  >
+                    <DescriptionListGroup className="description-list-spacer">
+                      <DescriptionListTerm>
+                        {t("steamingUnit")}
+                      </DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$1.49/hour"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                    <DescriptionListGroup className="description-list-spacer">
+                      <DescriptionListTerm>
+                        {t("dataTransfer")}
+                      </DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$0.09/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                    <DescriptionListGroup className="description-list-spacer">
+                      <DescriptionListTerm>{t("storage")}</DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$0.10/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                  </DescriptionList>
+                </FlexItem>
+              </Flex>
+              <Divider isVertical />
+              <Flex flex={{ default: "flex_1" }}>
+                <FlexItem>
+                  <TextContent>
+                    <p>
+                      A <strong>Streaming Unit</strong> determines the default
+                      maximum capacity of a Kafka Instance. The number Streaming
+                      Units that your Kafka Instance has determines its maximum
+                      capacity.
+                    </p>
+                  </TextContent>
+                </FlexItem>
+              </Flex>
+            </Flex>
+          </CardBody>
+        </Card>
+      </MarketingPageSection>
+      <MarketingPageSection>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h3">{t("kafkaInstanceCapacityTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Stack hasGutter>
+              <Text component={TextVariants.p}>
+                {t("kafkaInstanceCapacityDescription")}
+              </Text>
+              <KafkaInstanceCapacityTable />
+            </Stack>
+          </CardBody>
+          <CardFooter>
+            <TextContent>
+              <Text>
+                {
+                  <Trans
+                    i18nKey={"kafkaoverview:kafkaInstanceCapacityfooter"}
+                    components={{
+                      support: (
+                        <Link to="https://access.redhat.com/support"></Link>
+                      ),
+                      value: (
+                        <Link to="https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/aced8e5e-8229-4cb2-82f9-87a8caa24bb3"></Link>
```

Comment:
This wasn't noted in the mockup or design document, but ideally these links would have the following defined, similar to the links in the first section:

- `target="_blank"` to open in a new tab
- `<ExternalLinkAltIcon className="pf-u-ml-md" />` to indicate that it opens in a new tab

Reply:
this review comment is not addressed, still thinking about how to add icon with the JSON text